docs/codeql/writing-codeql-queries/introduction-to-ql.rst

```diff
@@ -17,13 +17,15 @@ QL also supports recursion and aggregates. This allows you to write complex recu
 Running a query
 ---------------
 
-You can try out the following examples and exercises using :ref:`CodeQL for VS Code <codeql-for-visual-studio-code>`, or you can run them in the `query console on LGTM.com <https://lgtm.com/query>`__. Before you can run a query on LGTM.com, you need to select a language and project to query (for these logic examples, any language and project will do).
+You can try out the following examples and exercises using :ref:`CodeQL for VS Code <codeql-for-visual-studio-code>`, or you can run them in GitHub Codespaces using the `CodeQL template <https://github.com/codespaces/new?template_repository=github/codeql-codespaces-template>`__. This will open a GitHub Codespaces environment preconfigured to run CodeQL queries.
```

Comment:
As discussed in Slack: I think we should pull the information about the template out of this section into its own 'pull-quote' or similar. This will let us:

1) make it very clear that it's in beta (so isn't quite GA-worthy yet)
2) make clear that it's only suitable to do the tutorials (including 'Find the thief' etc.) and a 'traditional' set up should be used to query real code.

Reply:
If we were to put the content mentioned in https://github.com/github/codeql/pull/11554/files#r1044617689 in a reusable, we could also display it in the language-agnostic tutorials 🤔